src/chapters/html/semantic-html.rst

```diff
@@ -0,0 +1,305 @@
+identify the number of sections. However, figuring out what each section does
+is more difficult. Is the ``<div>`` element on line 13 a figure caption for the
+``img``? Before we can say for sure, we would need to examine the content
+inside that element, or look at the final page in a browser.
+
+For the sample code above, we would want to include plenty of comments in the
+file to describe the purpose of each ``div`` element. After all, once we move
+on to our next project, another programmer might need to update the code for
+this page. We must make sure they understand what we created.
+
+.. admonition:: Tip
+
+   Making code work is important, but so is making it easier for others to
+   understand. Not every piece of code a programmer reads is something they
+   wrote.
+
+Besides comments, HTML also includes a way to use the tags themselves to
+describe the layout of a page. The code below builds exactly the same page as
+the first example. However, note how it helps answer the question, *What does
+this section do?*
+
+.. sourcecode:: html
+   :lineno-start: 6
+
+   <body>
+      <header>Content...</header>
+      <section>
+         <article>
+            Content...
+            <figure>
+               <img src="..." alt="Text...">
+               <figcaption>Content...</figcaption>
+            </figure>
+         </article>
+      </section>
+      <footer>Content...</footer>
+   </body>
+
+Aha! Finding the figure caption is a snap now, and we can see that we have a
+header and footer at the top and bottom of the page. It looks like the purpose
+of the ``section`` element is to organize related ``article`` and ``figure``
+elements, and the ``article`` tag name gives us a clue about the type of
+content that begins on line 10.
+
+HTML Writing Style
+------------------
+
+.. index:: ! semantic html
+
+Just like any other author, programmers write HTML with different style guides
+and ways of thinking. Two different coders can look at the same set of content
+and choose different ways to arrange it on the screen. However, both of them
+use the same set of HTML tags to do their work.
+
+**Semantic HTML** is not about the appearance of the webpage. Instead, it's
+about the specific meaning of the elements. Semantic HTML uses more descriptive
+tag names to help identify what each section of the code does. This helps
+programmers clarify the structure of a webpage.
+
+For example, developers can make a paragraph larger than a heading if they want
+to. By looking at the HTML, however, we can still identify what is the
+paragraph and what is the heading. The ``<p>`` and ``<h_>`` tags give us that
+information. This is the strength of semantic HTML. The tag names express
+meaning.
+
+Another benefit to semantic HTML is that programmers find it easier to
+visualize the results when they can put a specific name to a section of their
+code.
+
+Screen Readers and Search Engines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think that it is very important to bring up screen readers when talking about semantic HTML, but this explanation misses the mark for me. When discussing screen readers, it is important to distinguish between what screen readers do and how users utilize screen readers to navigate the page. Screen readers simply read out content; the program does not assign importance to anything.

Reply:
As I was writing this page, I realized that it was going to need some big revisions.  I'm also thrilled that it's you proofreading this content.

Let's set up a video chat so we can work on this at the same time.